src/main/scala/uk/gov/hmrc/common/message/model/Recipient.scala

```diff
@@ -109,19 +116,94 @@ object TaxIdentifierRESTV2Formats {
           Reads[TaxIdWithName] { _ =>
             JsError("The backend has rejected the message due to an unknown tax identifier.")
           }
-        case (Some(_), _) =>
+        case (Some(name), _) =>
           Reads[TaxIdWithName] { _ =>
-            JsError("The backend has rejected the message due to an unknown tax identifier.")
+            JsError(s"The backend has rejected the message due to an unknown tax identifier.")
```

Comment:
**Minor**: The `name` variable doesn't seem to be used and also the string `"The backend has rejected the message due to an unknown tax identifier."` doesn't seem to use interpolation

Reply:
@valydia Done, thanks